marlowe-chain-sync/libchainsync/Language/Marlowe/Runtime/ChainSync/NodeClient.hs

```diff
@@ -39,7 +39,7 @@ import Control.Monad.Event.Class (MonadInjectEvent, withEvent)
 import qualified Ouroboros.Network.Protocol.LocalStateQuery.Client as Q
```

Comment:
Changes in this file:

- Unification of connectToLocalNode type so it can be shared.